### PR TITLE
fix(DB/quest_template): Fix Quest: Rat Catching

### PR DESCRIPTION
**Pull request by Kosiak**
## Changes Proposed:
-  Fix wrong location name in CompletionLog.

## Issues Addressed:

- Closes [ Quest Rat Catching from POI in quest tracker #3139](https://github.com/chromiecraft/chromiecraft/issues/3139)

## Tests Performed:

- .q add 416
- .q comp 416

## How to Test the Changes:

1. Clean wow cache
2. Add quest (.q add 416)
3. Complete quest (.q comp 416)
4. Shows correct location for quest completion. (Return to [Mountaineer Kadrell](https://classic.wowhead.com/npc=1340/mountaineer-kadrell) at Thelsamar in Loch Modan.

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1647204744919951000.sql [hunk 1]
```diff
@@ -0,0 +1,3 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1647204744919951000');
+
+UPDATE `quest_template` SET `QuestCompletionLog` = 'Return to Mountaineer Kadrell at Thelsamar in Loch Modan.' WHERE `ID` = 416 AND `QuestSortID` = 38;
```
Comment (LukasVolgger) [inline data/sql/updates/pending_db_world/rev_1647204744919951000.sql:3]: Is `QuestSortID` necessary here if you filter by `ID`?

**Review by LukasVolgger — APPROVED** [review]

**Comment by LukasVolgger** [comment]
Thanks for the PR and the update. Please also make sure that you give us a source that confirms the correctness of your change. A test can then be omitted if everything else fits and it is only a minor change, as here.

**Comment by Kosiak** [comment]
> Thanks for the PR and the update. Please also make sure that you give us a source that confirms the correctness of your change. A test can then be omitted if everything else fits and it is only a minor change, as here.

Source: https://classic.wowhead.com/quest=416/rat-catching

**Comment by LukasVolgger** [comment]
> > Thanks for the PR and the update. Please also make sure that you give us a source that confirms the correctness of your change. A test can then be omitted if everything else fits and it is only a minor change, as here.
> 
> Source: https://classic.wowhead.com/quest=416/rat-catching

Ok, that's not quite what I meant. It's all about where you got the text from and that it was the same in wotlk. Classic is not the same as Wotlk. After my own research I could not find a source that confirms the correctness of your text. Do you have another source?

**Comment by Kosiak** [comment]
> > > Thanks for the PR and the update. Please also make sure that you give us a source that confirms the correctness of your change. A test can then be omitted if everything else fits and it is only a minor change, as here.
> > 
> > 
> > Source: https://classic.wowhead.com/quest=416/rat-catching
> 
> Ok, that's not quite what I meant. It's all about where you got the text from and that it was the same in wotlk. Classic is not the same as Wotlk. After my own research I could not find a source that confirms the correctness of your text. Do you have another source?

Unfortunately i couldn't find anything either before i submited pr. But current text in DB ( Return to Mountaineer Kadrell at  Grizzlepaw Ridge in Loch Modan.) is definitely wrong. As marker on map shows completion in Thelsamar. And all objectives for quest(TBC, Classic, wotlk(https://wotlkdb.com/?quest=416) got the same objective (Bring 12 Tunnel Rat Ears to Mountaineer Kadrell in Thelsamar.). Unfortunately quest is marked obsolete on live wowhead so i cant provide any source sorry. But logically why would you go to Grizzlepaw Ridge if quest starts and ends in Thelsamar. The change i made was only Grizzlepaw Ridge to Thelsamar the other parts of the text was in DB already.

**Comment by LukasVolgger** [comment]
That's enough as a source and I think it's understandable.  It's just that every change is documented as well as possible since it's a blizzlike project.

**Comment by Kosiak** [comment]
> That's enough as a source and I think it's understandable. It's just that every change is documented as well as possible since it's a blizzlike project.

Can you provide/suggest  me wowhead like site to use for this core specifically. i think using classic wowhead is not very reliable. It will help me to check it more properly before doing anything. Thank you

**Pull request merged**